src/graph.rs

```diff
@@ -822,6 +847,26 @@ impl<T> Graph<T> {
         VertexIter::new(collection)
     }
 
+    /// Returns the vertices hashmap 
+    ///
+    /// ## Example
+    /// ```rust
+    /// use graphlib::Graph;
+    ///
+    /// let mut graph: Graph<usize> = Graph::new();
+    ///
+    /// let v1 = graph.add_vertex(0);
+    /// let v2 = graph.add_vertex(1);
+    /// let v3 = graph.add_vertex(2);
+    /// let v4 = graph.add_vertex(3);
+    ///
+    /// let hashmap = graph.hashmap_vertices();
+    /// assert_eq!(hashmap.keys().len(), 4);
+    /// ```
+    pub fn hashmap_vertices(&self) -> &HashMap<Arc<VertexId>, (T, Arc<VertexId>)> {
```

Comment:
The graph shouldn't expose a vertices method.

Reply:
I dropped the `pub`. Now its private. Ok?